pkg/cmd/create/project.go

```diff
@@ -40,15 +39,38 @@ var projectCmd = &cobra.Command{
 
 		projectName, err := cmd.Flags().GetString("name")
 		utils.PrintError(err)
-
 		if projectName == "" {
-			utils.White_B.Print("\nEnter a project name: ")
-			fmt.Scanln(&projectName)
+			// prompt to ask project name
+			prompt := promptui.Prompt{
+				Label:     "Enter a project name",
+				AllowEdit: true,
+			}
+
+			result, err := prompt.Run()
+			if err != nil {
+				utils.Red.Printf("Error: %v\n", err)
+				return
+			}
+
+			projectName = result
 		}
 
 		_, err = apis.CreateProjectRequest(projectName, credentials)
-		utils.PrintError(err)
+		if err != nil {
+			utils.Red.Printf("❌ Error creating project: %v\n", err)
+		} else {
+			utils.White_B.Printf("Project '%s' created successfully!🎉\n", projectName)
```

Comment:
can we return the project ID also?

Reply:
as we are creating a project for the first time, we cannot ask it from user. therefore i am not sure how we can retrieve id the moment it is created.

Reply:
we will get a [createProjectReponse](https://github.com/litmuschaos/litmusctl/blob/49fc5b0ec766f3ea3553fedf3df042872f3b6ad8/pkg/apis/project.go#L34) which basically have `name` and `id` when we're sending a request for creating the project

Reply:
hey @Nageshbansal now create project shows project id too, thank you for your feedback!